docs/standard/memory-and-spans/index.md

```diff
@@ -30,6 +30,8 @@ Starting with .NET Core 2.1, .NET includes a number of interrelated types that r
 
 - <xref:System.MemoryExtensions?displayProperty=nameWithType>, a collection of extension methods for converting strings, arrays, and array segments to <xref:System.Memory%601> blocks.
 
+<xref:System.Span%601?displayProperty=nameWithType>, <xref:System.Memory%601?displayProperty=nameWithType>, and their readonly counterparts are designed to allow the creation of algorithms that avoid copying memory or allocating on the managed heap more than necessary. Creating them (either via `Slice` or their constructors) does not involve manipulating the underlying buffers: only the relevant references and offsets, which represent the "view" of the wrapped memory, are updated.
```

Comment:
Perhaps "manipulating" should be changed to "duplicating"? If there's confusion about how `Span<T>` and `Memory<T>` work, I don't think the confusion stems from consumers believing that the contents of the buffers will be altered. The confusion probably stems from when copies are made vs. when copies aren't made.

Reply:
Indeed, that's the point I hope to get across. I will replace it as per the suggestion.